README.md

```diff
@@ -1,13 +1,27 @@
 # Building Infrastructure for Data-Driven Research
 
-## Talk
+* **Speaker:** [Philipp Zumstein](https://www.bib.uni-mannheim.de/philipp-zumstein/) (![font-awesome_4-7-0_github_24_0_000000_none](https://cloud.githubusercontent.com/assets/5199995/23703699/94c8c3d2-0401-11e7-8b78-fd2785ce4c27.png) [@zuphilip](https://github.com/zuphilip), ![font-awesome_4-7-0_twitter_24_0_007dff_none](https://cloud.githubusercontent.com/assets/5199995/23703698/91c11176-0401-11e7-8e7d-bec431688160.png) [@zuphilip](https://twitter.com/zuphilip))
+* **Date:** March 15th, 2017
+* **Venue:** Social Science Data Lab, MZES, Mannheim
 
-* **Speaker:** [Philipp Zumstein](https://www.bib.uni-mannheim.de/philipp-zumstein/) (<img src="https://assets-cdn.github.com/images/modules/logos_page/GitHub-Mark.png" height="20px"/> [@zuphilip](https://github.com/zuphilip), <img src="https://cdn.pixabay.com/photo/2014/04/03/11/53/twitter-312464_640.png?attachment"/ height="18px"> [@zuphilip](https://twitter.com/zuphilip))
-* **Venue:** Social Science Data Lab, MZES, Mannheim, March 15th, 2017
 
 
 ## Abstract
 
+Most methods for data-driven research (including Big Data, Data Science, and Digital Humanities) work primarily on text data or numbers.
+However, there is also a lot of information which is only available in printed books or newspapers.
+These information has to be first digitized and then further processed to extract the text or data.
+The main focus of the talk is optical character recognition (OCR).
```

Comment:
I suggest _text recognition or, as it is traditionally called, optical character recognition (OCR)_.